content/templates/content/childrensprogram.html

```diff
@@ -0,0 +1,43 @@
+{% extends "content/base.html" %}
+
+{% block content %}
+
+ {% for program in childrensprograms %}
+ {% if forloop.counter0|divisibleby:2 %}
+ 	
+  {% if forloop.first %}
+ <div class="row">
+  	<div class="col-md-4">
+		<h1 class="childrenstitle text-center text-uppercase">{{program.pagetitle}}</h1>
+	</div>
+	<div class="col-md-8">
+	 <img class="center-block img-responsive programimg" src="{{program.img.url}}" alt="Children Studying">
```

Comment:
Talked with @RachellCalhoun on slack.
She is going to make the img field here not mandatory and add some help text in the admin explaining how it works